README.md

```diff
@@ -136,6 +163,7 @@ need to be provided in order for this action to successfully run using one of th
 | `targetPort`              | No       | The target port that the Container App will listen on. If not provided, this value will be "80" for Python applications and "8080" for all other supported platforms. |
 | `location`                | No       | The location that the Container App (and other created resources) will be deployed to. To view locations suitable for creating the Container App in, please run the following: `az provider show -n Microsoft.App --query "resourceTypes[?resourceType=='containerApps'].locations"` |
 | `environmentVariables`    | No       | A list of environment variable(s) for the container. Space-separated values in 'key=value' format. Empty string to clear existing values. Prefix value with 'secretref:' to reference a secret. |
+| `ingress`                 | No       | Possible options: external, internal, disabled. If set to "external" (default), the Container App will be visible from the internet or a VNET, depending on the app environment endpoint configured. If set to "internal", the Container App will be visible from within the app environment only. If set to "disabled", ingress will be disabled for this Container App and will not have an HTTP or TCP endpoint. |

```

Comment:
Just to be sure I get it: this one couldn't be set through the YAML and it was needed to surface it as a standalone parameter?

Reply:
I believe that everything in this Additional Arguments section falls under the `non-identity` category or arguments, that is, they will be ignored in favor of whatever is configured in the [YAML](https://learn.microsoft.com/en-us/azure/container-apps/azure-resource-manager-api-spec?tabs=yaml#examples)

Reply:
Both comments are correct -- ingress is a non-identifier property for a Container App. My reasoning for adding this was that the ingress concept itself felt like an important enough property on a Container App that it warranted having its own argument that users could set when using the non-YAML scenario. However, with the YAML file encapsulating this property, I'm OK with scrapping it and pointing users to the YAML scenario if they need that additional layer of configuration (which would be our current response if they were trying to update any other non-identifier property of the Container App). What do you think?

Reply:
I think your instinct was probably good: let's do what makes the customers the happiest. :)
Thanks both!

Reply:
+1 for your current approach